test/Api.IntegrationTest/SecretsManager/Controllers/AccessPoliciesControllerTest.cs

```diff
@@ -35,37 +38,96 @@ public AccessPoliciesControllerTest(ApiApplicationFactory factory)
 
     public async Task InitializeAsync()
     {
-        var ownerEmail = $"integration-test{Guid.NewGuid()}@bitwarden.com";
-        var tokens = await _factory.LoginWithNewAccount(ownerEmail);
-        var (organization, _) =
-            await OrganizationTestHelpers.SignUpAsync(_factory, ownerEmail: ownerEmail, billingEmail: ownerEmail);
+        _email = $"integration-test{Guid.NewGuid()}@bitwarden.com";
+        await _factory.LoginWithNewAccount(_email);
+        _organizationHelper = new SecretsManagerOrganizationHelper(_factory, _email);
+    }
+
+    public Task DisposeAsync()
+    {
+        _client.Dispose();
+        return Task.CompletedTask;
+    }
+
+    private async Task LoginAsync(string email)
+    {
+        var tokens = await _factory.LoginAsync(email);
         _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", tokens.Token);
-        _organization = organization;
     }
 
-    public Task DisposeAsync() => Task.CompletedTask;
+    [Theory]
+    [InlineData(false, false)]
+    [InlineData(true, false)]
+    [InlineData(false, true)]
+    public async Task CreateProjectAccessPolicies_SmNotEnabled_NotFound(bool useSecrets, bool accessSecrets)
+    {
+        var (org, _) = await _organizationHelper.Initialize(useSecrets, accessSecrets);
+        await LoginAsync(_email);
 
-    [Fact]
-    public async Task CreateProjectAccessPolicies()
+        var initialProject = await _projectRepository.CreateAsync(new Project
```

Comment:
Personal opinion, but unless we update the project we should name it `project` instead of `initialProject`.

```suggestion
        var project = await _projectRepository.CreateAsync(new Project
```

Reply:
Resolved in https://github.com/bitwarden/server/pull/2628/commits/79bce819b50d75b5ce021b4b4dfad21dc9c97cd8